src/test/java/com/iota/iri/utils/dag/DAGHelperTest.java

```diff
@@ -98,13 +98,13 @@ public void testTraverseApprovees() throws Exception {
             processed.add(t);
         }, set);
         
-        assertEquals(2, processed.size());
+        assertEquals("", 2, processed.size());
```

Comment:
Why empty?
I suppose you decide that the default message is good enough

Still, lets be get into a habit of adding a meaningful message.